gcloud-java-compute/src/main/java/com/google/gcloud/compute/Compute.java

```diff
@@ -1586,6 +1634,110 @@ public static ImageListOption fields(ImageField... fields) {
   }
 
   /**
+   * Class for specifying disk get options.
+   */
+  class DiskOption extends Option {
+
+    private static final long serialVersionUID = -4354796876226661667L;
+
+    private DiskOption(ComputeRpc.Option option, Object value) {
+      super(option, value);
+    }
+
+    /**
+     * Returns an option to specify the disk's fields to be returned by the RPC call. If this option
+     * is not provided, all disk's fields are returned. {@code DiskOption.fields} can be used to
+     * specify only the fields of interest. {@link Disk#diskId()},
+     * {@link DiskConfiguration#diskType()} and {@link SnapshotDiskConfiguration#sourceSnapshot()}
+     * or {@link ImageDiskConfiguration#sourceImage()} are always returned, even if not specified.
+     */
+    public static DiskOption fields(DiskField... fields) {
+      return new DiskOption(ComputeRpc.Option.FIELDS, DiskField.selector(fields));
+    }
+  }
+
+  /**
+   * Class for specifying disk list options.
+   */
+  class DiskListOption extends Option {
+
+    private static final long serialVersionUID = -5148497888688645905L;
+
+    private DiskListOption(ComputeRpc.Option option, Object value) {
+      super(option, value);
+    }
+
+    /**
+     * Returns an option to specify a filter on the disks being listed.
+     */
+    public static DiskListOption filter(DiskFilter filter) {
+      return new DiskListOption(ComputeRpc.Option.FILTER, filter.toPb());
+    }
+
+    /**
+     * Returns an option to specify the maximum number of disks returned per page. {@code pageSize}
+     * must be between 0 and 500 (inclusive). If not specified 500 is used.
+     */
+    public static DiskListOption pageSize(long pageSize) {
+      return new DiskListOption(ComputeRpc.Option.MAX_RESULTS, pageSize);
+    }
+
+    /**
+     * Returns an option to specify the page token from which to start listing disks.
+     */
+    public static DiskListOption pageToken(String pageToken) {
+      return new DiskListOption(ComputeRpc.Option.PAGE_TOKEN, pageToken);
+    }
+
+    /**
+     * Returns an option to specify the disk's fields to be returned by the RPC call. If this option
+     * is not provided, all disk's fields are returned. {@code DiskListOption.fields} can be used to
```

Comment:
Could apply same nits as above (disk's --> disk, add "either" after "and" in second to last line)